GiniVision/Tests/HelpMenuViewControllerTests.swift

```diff
@@ -93,10 +93,13 @@ final class HelpMenuViewControllerTests: XCTestCase {
         
         XCTAssertEqual(itemText, cell.textLabel?.text,
                        "cell text in the first row should be the same as the first item text")
+        
         // Compare cgColors because since iOS 13 cell.backgroundColor has the UIDynamicSystemColor type
-        XCTAssertEqual(cellBackgroundColor.cgColor,
-                       cell.backgroundColor!.cgColor,
-                       "cell background color should always be white")
+        if #available(iOS 13, *) {
```

Comment:
Shouldn't there be an `else` branch to assert this also on older versions?

Reply:
This is a little flaky as for instance on iOS 10 the colour is in a different colour space and the comparison fails while on iOS 11 `cell.backgroundColor` is nil and the test crashes. I have another PR in preparation that changes the test sim to iPhone 11, so the test will run.